### ISSUE TITLE
GoogleOptimizer throws exception when adding product via API

### ISSUE THREAD

**Issue by matthewwiggins** [magento/magento2#2315]
If Google Contents Experiments is enabled, adding a product using the `V1/products/` endpoint causes GoogleOptimizer/Observer/AbstractSave.php line 117 to throw `InvalidArgumentException`. With contents experiments disabled the product add completes successfully.

Example valid request json:

```
    "product": {
        "sku": "abc1",
        "name": "abc",
        "visibility": 4,
        "type_id": "simple",
        "price": 3.62,
        "status": 1,
        "attribute_set_id": 4
    }
```

**Comment by katmoon** [comment]
Hi @matthewwiggins 

Thank you for the reporting issue and sorry for the delay.
I have created internal ticket `MAGETWO-50417` for this issue.

Thanks,
Kateryna.

**Comment by mslabko** [comment]
Hello @matthewwiggins 
Issue was fixed on develop branch (https://github.com/magento/magento2/commit/d58c63eb5915d6c460ccf3526cce1f8828d3f4ae) and will be available in 2.1 version
Thank you for reporting!
